CHANGELOG.md

```diff
@@ -47,7 +47,25 @@ Contributors:
 - [@danielefrigo](https://github.com/danielefrigo) ([#3547](https://github.com/dbt-labs/dbt/pull/3547))
 
 
-## dbt 0.20.1 (Release TBD)
+## dbt 0.20.2 (Release TBD)
+
+### Under the hood
```

Comment:
I don't think this changelog entry is in the correct section since we are adding this to `0.21.0` and not `0.20.2`

Reply:
Nvm I see you backported to that as well